backend/check_stac_metadata/task.py

```diff
@@ -36,10 +36,11 @@ def lambda_handler(event: JsonObject, _context: bytes) -> JsonObject:
                 "type": "object",
                 "properties": {
                     DATASET_ID_KEY: {"type": "string"},
+                    DATASET_PREFIX_KEY: {"type": "string"},
```

Comment:
As discussed, this property isn't used by this task, so it shouldn't be in the schema. Ditto in other tasks.